ERROR: test/analyze/exhaustive_return/inexhaustive.myte:3:16 All branches of function foo must end in a return statement
 3 | fun foo(): int {}
   |                 ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:7:0 All branches of function foo must end in a return statement
 7 | }
   | ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:12:2 All branches of function foo must end in a return statement
 12 |   }
    |   ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:18:2 All branches of function foo must end in a return statement
 18 |   }
    |   ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:26:2 All branches of function foo must end in a return statement
 26 |   }
    |   ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:32:2 All branches of function foo must end in a return statement
 32 |   } else {
    |   ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:39:0 All branches of function foo must end in a return statement
 39 | }
    | ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:43:0 All branches of function foo must end in a return statement
 43 | }
    | ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:46:18 All branches of function bar must end in a return statement
 46 |   fun bar(): int {}
    |                   ^

ERROR: test/analyze/exhaustive_return/inexhaustive.myte:52:20 All branches of function baz must end in a return statement
 52 |     fun baz(): int {}
    |                     ^

ERROR: test/analyze/exhaustive_return/statement_after_return.myte:6:2 Unreachable statement after return
 6 |   2;
   |   ^^

ERROR: test/analyze/exhaustive_return/statement_after_return.myte:11:2 Unreachable statement after return
 11 |   return 2;
    |   ^^^^^^^^^

ERROR: test/analyze/exhaustive_return/statement_after_return.myte:21:2 Unreachable statement after return
 21 |   3;
    |   ^^